import java.util.Scanner;

public class pyramidnumbers {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        for (int i = 0; i < n; i++) {
            for ( int j = 1; j <= n - i; j++) {
                System.out.print("_");
            }
            for (int k = 0; k < i + 1; k++) {
                System.out.print("");
            }
            System.out.print("\n");
        }
        sc.close();
    }
}
